test/python/algorithms/test_measure_error_mitigation.py

```diff
@@ -66,8 +72,12 @@ def test_measurement_error_mitigation_with_diff_qubit_order(self):
         qc2.measure(1, 0)
         qc2.measure(0, 1)
 
-        # this should run smoothly
-        quantum_instance.execute([qc1, qc2])
+        if fitter_cls == TensoredMeasFitter:
+            self.assertRaises(QiskitError, quantum_instance.execute, [qc1, qc2])
```

Comment:
I'm a bit confused about this one -- shouldn't execute work with the `TensoredMeasFitter`?

Reply:
"TensoredMeasFitter doesn't support subset_fitter"  is the exception raised. I changed the unit test to make it more explicit.